### ISSUE TITLE
Do they (two learning rate schedule NoamLR and WarmupLR) have much difference during training?

### ISSUE THREAD

**Issue by yjiangling** [mobvoi/wenet#181]
Have you test the difference for this two learning rate schedule (NoamLR and WarmupLR)? When I use NoamLR to train CTC/AED joint model, it seems that it's quite hard to train it.

**Comment by robin1001** [comment]
We didn't try NoamLR.

**Comment by yjiangling** [comment]
> We didn't try NoamLR.

OK，thanks a lot for the reply. BTW，did you meet the problem of can not decrease the loss during training？How to solve it？
